### DISCUSSION TITLE
Details/Roadmap of "Google Docs"-like collaborative editor?

### DISCUSSION THREAD

**Discussion by Lauler** [quarto-dev/quarto-cli#778] [Q&A]
I read the quote about a planned collaborative editor in the Nature article and was curious to find out whether (or when) you are going to share details regarding those plans. In my opinion a collaborative online editor would be an absolute killer feature for a publishing system like Quarto. It would be a great boost to our ability to collaborate on blogs/books/articles with co-authors who aren't as technically savvy. 

I imagine it might do wonders for Quarto's adoption similar to the effect Overleaf/ShareLatex had in the LaTeX ecosystem.

Any details on when we might hear more about this tool?

**Comment by jjallaire** [comment]
No additional details other than to say that we understand how important this will be to Quarto's adoption so won't let too much grass grow under out feet before working on it in earnest!

**Comment by rgaiacs** [comment]
@Lauler can you share the DOI or link to the Nature article?

**Reply by Lauler** [reply]
Yes. Here it is: https://www.nature.com/articles/d41586-022-00563-z
